src/components/Cards.tsx

```diff
@@ -13,47 +17,106 @@ interface ICardsProps {
 }
 
 export default function Cards({ cards, onDelete }: ICardsProps) {
+  const theme = useTheme();
   return (
-    <div className="cards-container">
-      {cards.map((card, index) => (
-        <div key={index} className="card">
-          <table>
-            <tbody>
-              <tr>
-                <td>{CHECK_AND_RADIO[0].label}</td>
-                <td>
-                  {card.levelOfAI.map((level, levelIndex) => (
-                    <span key={levelIndex} className="value-block">
-                      {level}
-                    </span>
-                  ))}
-                </td>
-              </tr>
-              <tr>
-                <td>{CHECK_AND_RADIO[1].label}</td>
-                <td>
-                  {card.whereAIIsUsed.map((use, useIndex) => (
-                    <span key={useIndex} className="value-block">
-                      {use}
-                    </span>
-                  ))}
-                </td>
-              </tr>
-              <tr>
-                <td>{CHECK_AND_RADIO[2].label}</td>
-                <td>
-                  <span className="value-block">{card.TypeOfAI}</span>
-                </td>
-              </tr>
-              <tr>
-                <td>{RANGE[0].label}</td>
-                <td>{card.rateAIIntelligence}</td>
-              </tr>
-            </tbody>
-          </table>
-          <button onClick={() => onDelete(index)}>Delete</button>
-        </div>
-      ))}
-    </div>
+    <Box
+      sx={{
+        display: "flex",
+        flexDirection: "column",
+        gap: 6,
+      }}
+    >
+      <TransitionGroup>
+        {cards.map((card, index) => (
+          <CSSTransition key={index} timeout={500} classNames="card">
+            <Box
+              sx={{
+                flexDirection: "column",
+                padding: 2,
+                border: `2px solid ${theme.palette.text.primary}`,
+                borderRadius: "5px",
+                position: "relative",
+                marginBottom: 6,
+                "&::after": {
+                  content: '""',
+                  position: "absolute",
+                  bottom: theme.spacing(1),
+                  left: {
+                    xs: theme.spacing(1),
+                    sm: theme.spacing(2),
+                    md: theme.spacing(4),
+                  },
+                  right: {
+                    xs: theme.spacing(0.7),
+                    sm: theme.spacing(1),
+                    md: theme.spacing(2),
+                  },
+                  height: "2px",
+                  backgroundColor: theme.palette.text.primary,
+                },
+              }}
+            >
+              <h3>
+                {CHECK_AND_RADIO[0].label}:
+                {card.levelOfAI.map((option, index) => (
+                  <Chip
+                    key={index}
+                    label={option}
+                    style={{ margin: theme.spacing(0, 0.5) }}
```

Comment:
why style not sx?

Reply:
I did as in the example, and it was like this. And then in the end this part remained